arc-api/server/models/subscription.js

```diff
@@ -192,6 +206,11 @@ module.exports = function(Subscription) {
       },
       json: true
     }, function(err, res, body) {
+      if (err && mode !== 'online') {
```

Comment:
This is the correct logic. If an error happens from the REST call, the error will be reported under `online` mode. Otherwise it will fall back to `offline` loading.